packages/rest/src/rest.application.ts

```diff
@@ -97,10 +97,14 @@ export class RestApplication extends Application implements HttpServerLike {
 
   /**
    * Bind a body parser to the server context
-   * @param bodyParserClass
+   * @param parserClass Body parser class
+   * @param address Optional binding address
    */
-  bodyParser(bodyParserClass: Constructor<BodyParser>) {
-    this.restServer.bodyParser(bodyParserClass);
+  bodyParser(
+    bodyParserClass: Constructor<BodyParser>,
+    address?: BindingAddress<BodyParser>,
```

Comment:
Can we use `key` instead of `address` for consistency with other places please? Most notably `Context` class uses `key` in most of places.

Reply:
I just checked. The `Context` class `BindingAddress` for most cases.

If you refer to `app.controller(controllerClass, name)` or similar methods, I intentionally make it different so that we can use the well-know binding address to override built-in body parsers. 

BTW, `export const REQUEST_BODY_PARSER_JSON` in `keys.ts` is by design to support the acceptance test of overriding built-in body parsers.